src/main/java/com/rarchives/ripme/ripper/rippers/HentaifoundryRipper.java

```diff
@@ -94,16 +94,18 @@ public Document getNextPage(Document doc) throws IOException {
                 logger.info("Couldn't find user & image ID in " + thumb.attr("href"));
                 continue;
             }
+            String[] titleSplit = thumb.attr("herf").split("/");
+            String title = titleSplit[titleSplit.length -1];
             String user = imgMatcher.group(1),
                 imageId = imgMatcher.group(2);
             String image = "http://pictures.hentai-foundry.com//";
             image += user.toLowerCase().charAt(0);
-            image += "/" + user + "/" + imageId + ".jpg";
+            image += "/" + user + "/" + imageId + "/" + user + "-" + imageId + title + ".png";
```

Comment:
> `.png`

Strictly PNGs now? String JPG before? Or is this like imgur where file ext on the url doesn't matter and it serves up something reasonable?